docs/PR_GUIDE.md

```diff
@@ -0,0 +1,92 @@
+
+- PR should be small and reflect only one idea, a feature, a bugfix, or a refactoring. In most cases a PR of 100 lines
+  or less is considered as a PR of normal size and a PR of 1000 lines as big one
+- If a PR implements two different features, refactoring, or bugfixes, it should be split into several PRs
+- The description field of every PR should contain a description, links to a ticket or/and links to the confluence
+- If a PR implements a complex math task, the description should contain a link to the proof of the solution or a link
+  to a trusted source
+- All PRs should contain a link to the corresponding ticket. The exception is refactoring if there’s nothing to test
+  after it. The reason for that is that every feature or bugfix should be tested by testers after it’s implemented
+- Every commit of all PRs should be compilable under all platforms. All tests should pass. So if changing of code
+  breaks unit or integration tests these tests should be fixed in the same commit
+- Every commit should reflect a completed idea and have an understandable comment. Review fixes should be merged into
+  one commit
+- New functionality and unit or integration tests for it should be developed in the same PR
+- When some source files are changed and then some other source files based on it are auto generated, they should be
+  committed in different commits. For example, one commit changes strings.txt and another one contains generated files
+- Most commits and PRs should have prefixes in square brackets depending on the changed subsystem. For example,
+  [routing], [generator], or [android]. A commits and a PRs may have several prefixes
+- If a PR should not be merged after review, the caption of it should be started with [DNM] prefix or the PR should be
+  published as a draft. DNM means do not merge
+- All commits and PR captions should be written in English. PR description and PR comments may be written in other
+  languages
+- All PRs should be approved by at least two reviewers. There are some exceptions: (1) simple PRs to some tools or tests;
+  (2) situation when it’s impossible to find another competent reviewer. In any case a PR should be reviewed by
+  at least one reviewer
+- An author should not merge their PR. One of the reviewers should merge it
+- If somebody left a comment in PR the author should wait for their approval
+
+
+## Doing good code review:
+
+- All comments in PR should be polite and concern the PR
+- If a reviewer criticizes a PR they should suggest a better solution
+- Reviewer's solution may be rejected by the author if the review did not give arguments why it should be done 
+- Comments in PR should be not only about things which are worth redeveloping but about good designs as well
+- It’s better to ask developer to make code simpler or add comments to code base than to understand the code through
+  developer's explanation
+- It’s worth writing comments in PR which help a PR author to study something new
```

Comment:
s/to study/study/
or learn

Reply:
Are you sure about skipping "to"? "... the “to” is often left out, especially in informal or spoken English."

https://www.oxfordlearnersdictionaries.com/definition/american_english/help_1

`It’s worth writing comments in PR which help a PR author to learn something new`

I changed "to study" to "to learn".